pkg/webhook/resources/virtualmachine/mutator.go

```diff
@@ -373,3 +378,27 @@ func (m *vmMutator) addNodeSelectorTerms(defaultNamespace string, nodeSelector *
 
 	return nil
 }
+
+func (m *vmMutator) patchTerminationGracePeriodSeconds(vm *kubevirtv1.VirtualMachine, patchOps types.PatchOps) (types.PatchOps, error) {
```

Comment:
seems there is no global config on the kubevirt CR?

Reply:
Yeah, I don't find global config on KubeVirt CR.